crates/rolldown/src/module_loader/module_task.rs

```diff
@@ -231,12 +239,13 @@ impl ModuleTask {
 
     let module_info = Arc::new(module.to_module_info());
     self.ctx.plugin_driver.set_module_info(&module.id, Arc::clone(&module_info));
-    self.ctx.plugin_driver.module_parsed(module_info).await?;
+    self.ctx.plugin_driver.module_parsed(Arc::clone(&module_info)).await?;
+    self.ctx.plugin_driver.mark_context_load_modules_loaded(&module.id).await?;
```

Comment:
Would be good if we don't need to await here. Is it possible just `tokio::spawn` it?

Reply:
Not sure why don't need to await here. We need to callback the js side function, here need to using `awiat`. If the callback hasn't exit, it is also correct.

Reply:
Let's assume if we don't await here, what bad thing might happens?

- The task is being `spawn`, so the js callback would be executed in another thread. And it gets correct result, because ` self.ctx.plugin_driver.set_module_info(&module.id, Arc::clone(&module_info));` is already called.

Reply:
Yeah. It could be done moving to other thread. But looks like it hasn't profit from it, it maybe seize the thread from module loader task.